MatomoTracker.php

```diff
@@ -214,6 +219,38 @@ public function setGenerationTime($timeMs)
         return $this;
     }
 
+    /**
+     * Sets timings for various performance metrics.
```

Comment:
@sgiehl btw might be good to add a little documentation to each parameter like whether they are ms, etc.

Reply:
done